README.md

```diff
@@ -59,9 +59,9 @@ Conditions determine whether a flag is enabled or disabled by comparing a define
 
 ## Usage
 
-### Overview
+### Quickstart
 
-To use Wagtail-Flags you first need to define the flag, use the flag in code, and define conditions for the flag to be enabled.
+To use Wagtail-Flags you first need to define the flag, use the flag, and define conditions for the flag to be enabled.
 
 First, define the flag in Django `settings.py`:
 
```

Comment:
This wasn't changed in this PR, but as part of reviewing this I noticed a missing `)` at the end of [line 94](https://github.com/cfpb/wagtail-flags/pull/22/files#diff-04c6e90faac2675aa89e2176d2eec7d8R94). Instead of

```py
url(r'^/mypage$', TemplateView.as_view(template_name='mytemplate.html'),
```

it should be

```py
url(r'^/mypage$', TemplateView.as_view(template_name='mytemplate.html')),
```